applications/root.md

```diff
@@ -0,0 +1,367 @@
+
+The code of the preferundum pallet is available in [Matthiew's github](https://github.com/Matthiewm23/Preferundum/tree/main/pallets/voting/src) ; we are developing it, it is not fully tested but the logic is in. 
+
+## Development Roadmap :nut_and_bolt:
+
+- Milestone 1. Back-End
+- Milestone 2. Front-End
+
+### Overview
+
+- **Total Estimated Duration:** 5 months
+- **Full-Time Equivalent (FTE):**  6 FTE
+- **Total Costs:** 96 500 USD
+
+### Milestone 1 - Back-End — stated.tech - Preferendum Pallet - 40k
+
+- **Estimated Duration:** 3 month
+- **FTE:**  2
+- **Costs:** 40,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can create a preferundum, edit one, add possibility, vote, participate in the proposal |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish some **articles** that explains the philosophy, the interest and the roadmap |
+| 0f. | Research on architecture, pallet integrations, storage, preferendums, voting systems, UX | done |
+| 1. | Substrate module: Proposal | Our proposal making process is fully Org-agnostic: Multisigs, proxy-based organizations or any single address are welcome to create preferendums. Whenever there is a join, we will add the address account to the proposal. |
+| 2. | Substrate module: Quorum | We're building a custom quorum specifically designed for Preferendums. This quorum is a hybrid between an "optimistic" and "pessimistic" quorum that enables to take decisions in a way that suits each phase of the proposal process. the first phase uses an optimistic quorum that only takes into account vetoes, while the second phase which is the voting phase, uses a pessimistic quorum. Quorums are needed to decide when to vote or when to end a phase. There will be function where you can modify the maximum number of people that can join the team, and modify it over time. |
+| 3. | Substrate module: Preferundum | The pallet preferundum will host functions described in Milestone 2. It will be linking the hashmaps to IPFS and store them on the blockchain. Vetoes and joins are managed by the Quorum module (optimistic). |
+| 4. | Substrate module: Vote | For the logic of the vote, we will allow for a more democratic voting by giving each member of the proposal the same voting power. Using the **range-normalization** method, voters will decide percentages of their will to support this different alternatives. Voting is started and stopped according to quorum module (pessimistic).|
```

Comment:
For this first implementation, range normalization was designed to be as democratic as possible so it won't involve any token holding in the process. For each subject, participants would have a 100% voting power that would be redistributed to each alternative. Indeed, each preferendum is independent of the other and people within an org have the same voting power. Please also note that only participants (writers) will be able to vote on their own proposal when the voting phase will come so that it generates a consensus between all writers before sending it to a treasury referendum, namely Opengov (which is subject to a larger consensus and can still reject the proposal).

Reply:
Are preferendums by default multiple choice? Otherwise I still don't understand how range normalization works here. For example, if it was a simple yes/no vote, there are no alternatives to give weight to.

Reply:
If there is only one alternative on a specific subject, the voting process will not count them as part of a preferendum, but this alternative will be subject to a potential veto during the writing phase (if it's a spam or irrelevant). If vetoes pass via quorum, then the subject will not be taken into account during the voting phase.

Reply:
For example:

**WRITING PHASE:**

Proposal XYZ

Subject 1 by alice:
- Alternative 1 by alice

Subject 2 by alice:
- Alternative 1 by alice
- Alternative 2 by bob

Subject 3 by bob:
- Alternative 1 by bob
- Alternative 2 by bob
- Alternative 3 by alice

Subject 4 by Charlie:
- Alternative 1 by Charlie (VETOED)

**VOTING PHASE**

Proposal XYZ

Subject 1 by alice: 
- Alternative 1 by alice: AGREED

Subject 2 by alice:
- Alternative 1 by alice: 30%
- Alternative 2 by bob: 70%

Subject 3 by bob:
- Alternative 1 by bob: 30%
- Alternative 2 by bob: 30%
- Alternative 3 by alice: 40%

Subject 4 is canceled and thus invisible during voting.

**APPROVED PROPOSAL**

Subject 1 by alice
Alternative 1 by Alice

Subject 2 by alice
Alternative 2 by bob

Subject 3 by bob
Alternative 3 by alice

Reply:
@keeganquigley this might interest you as well regarding our mockups.